osu.Game/Rulesets/Mods/ModFlashlight.cs

```diff
@@ -47,9 +47,25 @@ public abstract class ModFlashlight<T> : ModFlashlight, IApplicableToDrawableRul
         public void ApplyToScoreProcessor(ScoreProcessor scoreProcessor)
         {
             Combo.BindTo(scoreProcessor.Combo);
+
+            // Default value of ScoreProcessor's Rank in Flashlight Mod should be SS+
+            scoreProcessor.Rank.Value = ScoreRank.XH;
```

Comment:
Note: This matches `ModHidden`, but eventually needs to be moved into `ScoreProcessor` itself.